users_file.c

```diff
@@ -0,0 +1,176 @@
+#include <stdio.h>
+#include <string.h>
+
+struct User {
+    int id;
+    char name[100];
+    int age;
+};
+
+void create_users() {
```

Comment:
No check for duplicate user IDs, meaning the same ID could be entered multiple times.